.github/workflows/ci-nix.yml

```diff
@@ -67,6 +67,48 @@ jobs:
       - name: Check `nix flake show`
         run: nix flake show .#
 
+  shell:
+    if: github.repository == 'fedimint/fedimint'
+    strategy:
+      matrix:
+        host:
+          - linux
+          - macos
+        include:
+          - host: linux
+            runs-on: buildjet-4vcpu-ubuntu-2004
+            build-in-pr: true
+            timeout: 30
+            run-tests: true
+          - host: macos
+            runs-on: macos-14
+            build-in-pr: false
+            timeout: 30
+            run-tests: false
+
+    name: "Dev Shell on ${{ matrix.host }}"
+    runs-on: ${{ matrix.runs-on }}
+    timeout-minutes: ${{ matrix.timeout }}
+
+    steps:
+      - uses: actions/checkout@v4
+        if: github.event_name != 'pull_request' || matrix.build-in-pr
+
+      - uses: cachix/install-nix-action@v25
+        if: github.event_name != 'pull_request' || matrix.build-in-pr
+        with:
+          nix_path: nixpkgs=channel:nixos-22.05
+      - uses: cachix/cachix-action@v14
+        if: github.event_name != 'pull_request' || matrix.build-in-pr
+        with:
+          name: fedimint
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+        continue-on-error: true
+
+      - name: Build dev shell
+        if: github.event_name != 'pull_request' || matrix.build-in-pr
```

Comment:
Looks like this takes ~1m in CI. Would it make sense to instead only run in the merge queue?

Reply:
I already disabled macos one in PRs, but I guess I don't really care that much about Linux one as well - as long as `master` doesn't have a broken dev shell it's OK. And it barely ever happens anyway.